sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceAsyncClient.java

```diff
@@ -55,18 +52,21 @@ public final class WebPubSubServiceAsyncClient {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<WebPubSubClientAccessToken> getClientAccessToken(GetClientAccessTokenOptions options) {
+        final ClientEndpointType clientEndpointType = options.getClientEndpointType();
+        final String path = clientEndpointType.equals(ClientEndpointType.MQTT)
```

Comment:
ClientEndpointType seems not committed to PR?

Reply:
Also I didn't see change to GetClientAccessTokenOptions.java

Reply:
Oh I'm seeing both changes are included in the PR already. Can you double check? 
https://github.com/Azure/azure-sdk-for-java/pull/40784/files#diff-378f75168506064970a2cbfd5ad1647468e502cba50e29073a0e099db491ef7c
https://github.com/Azure/azure-sdk-for-java/pull/40784/files#diff-4a2e7e8521c6c5d25ae24a7f3d8b78066f189dbd70d04c8e1344fc737f805c71